mablhq/github-run-tests-action
# [IST-253] Update Dependencies, Cleanup Unit Tests

- Move to `src/ test/` layout
- Use `jest` over `mocha`
- Tidy up tests
- Ensure `.npmrc` files can never be committed
- Bump dependencies to address CVE warnings
- `lodash`, `underscore`, `git-hosted-info`
- Fix E2E integration test
